examples/go/example-datacenter.yaml

```diff
@@ -1,17 +1,15 @@
-
 apiVersion: cassandraoperator.instaclustr.com/v1alpha1
```

Comment:
Why do we have a separate go folder for these examples? Just overwrite the existing ones

Reply:
Yep, fair point.